core/src/themes/ionic/ionic.theme.default.scss

```diff
@@ -10,85 +10,211 @@
+// - shade: A darker variant of the base color, used for pressed/active states
+// - tint: A lighter variant of the base color, used for ?
+// - foreground: The main color used for text and foreground elements
 
 // TODO(ROU-10778, ROU-10875): Sync the color names to the design system of
 // ios and md. This will allow us to have a single color map.
-$primary: #105cef;
-$secondary: initial;
-$tertiary: initial;
-$success: #1fbd3b;
-$warning: #e18300;
-$danger: #bf2222;
-$light: #f2f4fd;
-$medium: initial;
-$neutral: #626262;
-$dark: initial;
-
 $ionic-colors: (
   primary: (
-    base: $primary,
-    contrast: #fff,
-    shade: color.get-color-shade($primary),
-    tint: color.get-color-tint($primary),
-  ),
-  secondary: (
-    base: $secondary,
-    contrast: $secondary,
-    shade: color.get-color-shade($secondary),
-    tint: color.get-color-tint($secondary),
-  ),
-  tertiary: (
-    base: $tertiary,
-    contrast: $tertiary,
-    shade: color.get-color-shade($tertiary),
-    tint: color.get-color-tint($tertiary),
-  ),
-  success: (
-    base: $success,
-    contrast: #000,
-    shade: color.get-color-shade($success),
-    tint: color.get-color-tint($success),
+    bold: (
+      base: globals.$ion-bg-primary-base-default,
+      base-rgb: globals.$ion-bg-primary-base-default-rgb,
+      contrast: globals.$ion-text-inverse,
+      contrast-rgb: globals.$ion-text-inverse-rgb,
+      shade: globals.$ion-bg-primary-base-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-semantics-primary-600,
+      foreground: globals.$ion-text-primary,
+    ),
+    subtle: (
+      base: globals.$ion-bg-primary-subtle-default,
+      base-rgb: globals.$ion-bg-primary-subtle-default-rgb,
+      contrast: globals.$ion-text-primary,
+      contrast-rgb: globals.$ion-text-primary-rgb,
+      shade: globals.$ion-bg-primary-subtle-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-semantics-primary-200,
+      foreground: globals.$ion-text-primary,
+    ),
   ),
+  // info equivalent
+  secondary:
+    (
+      bold: (
+        base: globals.$ion-bg-info-base-default,
+        base-rgb: globals.$ion-bg-info-base-default-rgb,
+        contrast: globals.$ion-text-inverse,
+        contrast-rgb: globals.$ion-text-inverse-rgb,
+        shade: globals.$ion-bg-info-base-press,
+        // TODO this is not a verified value
+        tint: globals.$ion-semantics-info-700,
+        foreground: globals.$ion-text-info,
+      ),
+      subtle: (
+        base: globals.$ion-bg-info-subtle-default,
+        base-rgb: globals.$ion-bg-info-subtle-default-rgb,
+        contrast: globals.$ion-text-info,
+        contrast-rgb: globals.$ion-text-info-rgb,
+        shade: globals.$ion-bg-info-subtle-press,
+        // TODO this is not a verified value
+        tint: globals.$ion-semantics-info-200,
+        foreground: globals.$ion-text-info,
+      ),
+    ),
+  // TODO - no equivalent for tertiary, can we add purple as a color?
```

Comment:
Tertiary is the only one missing

Reply:
If we need to add something dummy for now, UX suggested Violet